lib/UnoCore/Targets/Android/com/fuse/App.java

```diff
@@ -88,6 +89,11 @@ public void onCreate(Bundle savedInstanceState)
 
         // call c++ for setup
         ActivityNativeEntryPoints.cppOnCreate(RootActivity);
+        
+        // reset window background for app switcher to be able to take a screenshot of app content
+        #if @(Project.Android.Splash.Enabled:IsSet)
+        RootActivity.getWindow().setBackgroundDrawableResource(android.R.color.transparent);
+        #endif
```

Comment:
I personally prefer `#` directives to not be indented, but that's not a big deal.

Reply:
Quick question, how do I check if that project variable has been set and that its "true"?

Reply:
Or maybe I should remove the "Enabled" property and just check if Android.Splash is set?

Reply:
I think you can `#if @(Project.Android.Splash.Enabled:Test(1, 0)`. That'll default to `0` when not set.

Reply:
You'll probably need the `Enabled` part, I think, or you'll have to check if any of `SplashBackgroundColor` or `SplashLogo` are set.

Reply:
Ok, lemme try that; I've got defaults for the background colour and logo